src/main/java/core/basesyntax/service/operations/OperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface OperationHandler {
+    void getOperation(List<FruitTransaction> transactions);

```

Comment:
The method name `getOperation` is not descriptive of the action it performs. Consider renaming it to reflect that it's handling an operation, such as `handle` or `execute`. Method names should be verbs or verb phrases that indicate what the method does.